clients/src/main/java/org/apache/kafka/clients/ClusterConnectionStates.java

```diff
@@ -479,7 +479,7 @@ private static class NodeConnectionState {
         private final String host;
         private final HostResolver hostResolver;
         private InetAddress lastAttemptedAddress;
-        private Logger log;
+        private final Logger log;
```

Comment:
We typically have the final fields grouped first.

Reply:
ack. will change.

Reply:
This is done in the latest commit.